libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -376,6 +376,48 @@ class AmazonTextractPDFParser(BaseBlobParser):
     """Send `PDF` files to `Amazon Textract` and parse them.
 
     For parsing multi-page PDFs, they have to reside on S3.
+
+    The AmazonTextractPDFLoader calls the
+    [Amazon Textract Service](https://aws.amazon.com/textract/)
+    to convert PDFs into a Document structure.
+    The loader does pure OCR at the moment, with more features like
+    layout support planned, depending on demand.
+    Single and multi-page documents are supported with up to 3000 pages
+    and 512 MB of size.
+
+    For the call to be successful an AWS account is required,
+    similar to the
+    [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+    requirements.
+
+    Besides the AWS configuration, it is very similar to the other PDF
+    loaders, while also supporting JPEG, PNG and TIFF and non-native
+    PDF formats.
+
+    ```python
+    from langchain.document_loaders import AmazonTextractPDFLoader
+    loader=AmazonTextractPDFLoader("example_data/alejandro_rosalez_sample-small.jpeg")
+    documents = loader.load()
+    ```
+
+    One feature is the linearization of the output.
+    When using the features LAYOUT, FORMS or TABLES together with Textract
+
+    ```python
+    from langchain.document_loaders import AmazonTextractPDFLoader
+    # you can mix and match each of the features
+    loader=AmazonTextractPDFLoader(
+        "example_data/alejandro_rosalez_sample-small.jpeg",
+        textract_features=["TABLES", "LAYOUT"])
+    documents = loader.load()
+    ```
+
+    it will generate output that formats the text in reading order and
+    try to output the information in a tabular structure or
+    output the key/value pairs with a colon (key: value).
+    This helps most LLMs to achieve better accuracy when
+    processing these texts.
```

Comment:
```suggestion
    AmazonTextractPDFLoader also supports linearization of the output, which maintains the structure of the input 
    document. For example, when using LAYOUT feature, the reading order of the text is maintained. Similarly, 
    when selecting FORMS feature, information is extracted into key-value pairs (key: value). With TABLES feature, 
    any tables present in the document are extracted into a tabular structure. These features helps the LLMs 
    process the information with better accuracy. Here is an example of using the TABLES and LAYOUT feature.
    
    ```python
    from langchain.document_loaders import AmazonTextractPDFLoader
    # you can mix and match each of the features
    loader=AmazonTextractPDFLoader(
        "example_data/alejandro_rosalez_sample-small.jpeg",
        textract_features=["TABLES", "LAYOUT"])
    documents = loader.load()
    ```
```